src/AppBundle/Model/User/Handler/CreateUserHandler.php

```diff
@@ -101,7 +104,12 @@ public function __invoke(CreateUserDTO $userDTO)
                 'field'  => 'pendingActivation.key',
             ];
 
-            $isUnique = count($this->validator->validate($activationKey, new UniqueProperty($options))) === 0;
+            $hash          = [
+                'recaptchaHash' => $userDTO->recaptchaHash,
+            ];
+
+            $isUnique       = count($this->validator->validate($activationKey, new UniqueProperty($options))) === 0;
+            $IsCaptchaValid = count($this->validator->validate($activationKey, new Recaptcha($hash))) === 0;
```

Comment:
can be reverted, during the evaluation with the command bus a middleware validates the DTO, this is only for the approval key generation.